### PR TITLE
Add Margined Redemption Rate Vaults

### PR DESCRIPTION
**Pull request by shapeshed**
This adds three new redemption rate vaults from Margined Protocol for Liquid Staked Assets for OSMO, TIA and ATOM. 

Strategy description documentation can be found here https://docs.margined.io/vaults/overview/redemption-rate-strategy

cc @JeremyParish69

### CODE REVIEW

**Comment by JeremyParish69** [comment]
We'll need to develop a Report Card for this as well. Could you please submit one?

**Comment by shapeshed** [comment]
> We'll need to develop a Report Card for this as well. Could you please submit one?

Links for each report card is included in the JSON. I think this is normally copied over but if there is anything needed let me know!

**Comment by JeremyParish69** [comment]
> > We'll need to develop a Report Card for this as well. Could you please submit one?
> 
> Links for each report card is included in the JSON. I think this is normally copied over but if there is anything needed let me know!

Apologies, I never remember to look there because we can't actually use those, I just need that info that's on them to create a new version.

**Comment by shapeshed** [comment]
Is there anything else need on this @JeremyParish69?

File: cms/earn/strategies.json [hunk 1]
```diff
@@ -2639,7 +2719,119 @@
...
+      "positionDenoms": [
+        {
+          "coinMinimalDenom": "factory/osmo15fqmdl8lfl9h0qflljd63ufw9j2m7xmsk3hu5vsn8xpta4hk5chqt7mddc/redemption-vault-uosmo",
+          "_comment": "OSMO Locust Redemption Vault Token"
+        }
+      ],
+      "rewardDenoms": [],
+      "categories": [
+        "Correlated",
+        "Blue Chip"
```
Comment (JeremyParish69) [inline cms/earn/strategies.json:2761]: Blue Chip is defines as top 50, so we can't have OSMO or TIA being categorized as "Blue Chip" anymore

File: cms/earn/strategies.json [hunk 2]
```diff
@@ -2639,7 +2719,119 @@
...
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/atom-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=634542877#gid=634542877",
```
Comment (JeremyParish69) [inline cms/earn/strategies.json:2813]: Please replace with this URL: https://docs.google.com/spreadsheets/d/1RlH1KT6EUJCY6ngJ_8TdPp9csF9T9iB8euyNlGSM17Y/edit?usp=sharing

File: cms/earn/strategies.json [hunk 3]
```diff
@@ -2639,7 +2719,119 @@
...
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/tia-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=1058195732#gid=1058195732",
```
Comment (JeremyParish69) [inline cms/earn/strategies.json:2777]: Please replace with this URL: https://docs.google.com/spreadsheets/d/1RlH1KT6EUJCY6ngJ_8TdPp9csF9T9iB8euyNlGSM17Y/edit?gid=1503172972#gid=1503172972

File: cms/earn/strategies.json [hunk 4]
```diff
@@ -2639,7 +2719,119 @@
...
+      "type": "Liquid Staking",
+      "method": "margined-locust-redemption",
+      "link": "https://trade.margined.io/vaults/osmo-redemption-rate",
+      "tvl": "https://vaults.margined.io/api/v1/${id}/tvl",
+      "apr": "https://vaults.margined.io/api/v1/${id}/apr",
+      "balance": "https://vaults.margined.io/api/v1/${id}/balance/${address}",
+      "geoblock": "https://vaults.margined.io/api/v1/${id}/geoblock",
+      "lockDuration": "PT0S",
+      "riskLevel": 1.00,
+      "riskReportUrl": "https://docs.google.com/spreadsheets/d/17WPa1fRkCzjZeUawp7g6zoEviQT_pJrPVGaMk9JEKsc/edit?gid=1475100361#gid=1475100361",
```
Comment (JeremyParish69) [inline cms/earn/strategies.json:2741]: Please replace with this URL: https://docs.google.com/spreadsheets/d/1RlH1KT6EUJCY6ngJ_8TdPp9csF9T9iB8euyNlGSM17Y/edit?gid=2069375532#gid=2069375532

**Comment by JeremyParish69** [comment]
> Is there anything else needed on this @JeremyParish69?

Apologies for the delay. Once we address the comments, we should be good to go.

**Comment by shapeshed** [comment]
@JeremyParish69 thanks for the review, updated!

**Pull request merged**